src/Stratis.Bitcoin.IntegrationTests/MinerTests.cs

```diff
@@ -152,7 +153,9 @@ public class TestContext
 
                 var peerAddressManager = new PeerAddressManager(nodeSettings.DataFolder, loggerFactory);
                 var peerDiscovery = new PeerDiscovery(new AsyncLoopFactory(loggerFactory), loggerFactory, Network.Main, networkPeerFactory, new NodeLifetime(), nodeSettings, peerAddressManager);
-                var connectionManager = new ConnectionManager(dateTimeProvider, loggerFactory, this.network, networkPeerFactory, nodeSettings, new NodeLifetime(), new NetworkPeerConnectionParameters(), peerAddressManager, new IPeerConnector[] { }, peerDiscovery);
+                var connectionSettings = new ConnectionManagerSettings();
+                connectionSettings.Load(nodeSettings);
+                var connectionManager = new ConnectionManager(dateTimeProvider, loggerFactory, this.network, networkPeerFactory, nodeSettings, new NodeLifetime(), new NetworkPeerConnectionParameters(), peerAddressManager, new IPeerConnector[] { }, peerDiscovery, connectionSettings);
```

Comment:
If we pass `nodeSettings` to ConnectionManager then we probably dont need to pass `connectionSettings`  as well right?

Reply:
Ah I realise NodeSettings does not have  ConnectionManagerSettings anymore

Reply:
Ok.